lark/grammars/python.lark

```diff
@@ -111,7 +114,8 @@ suite: simple_stmt | _NEWLINE _INDENT stmt+ _DEDENT
 ?comparison: expr (comp_op expr)*
 star_expr: "*" expr
```

Comment:
```suggestion
star_expr: "*" expr | assign_expr
```
It looks like a right place for `assign_expr`

Reply:
As [here](https://docs.python.org/3/reference/expressions.html#expression-lists)

Reply:
No, this would make `a := 5 > 6` mean something different that the original grammar from what I can see.

It has to be on a similar level to `lambdadef` from what I can see.